packages/storybook/stories/wc-helpers.jsx

```diff
@@ -174,46 +170,47 @@ export function CustomEventsDescription({ data }) {
 }
 
 export function ComponentDescription({ data }) {
+  if (!data) return null;
+
   return <div className="vads-u-margin-top--2">{data}</div>;
 }
 
 /**
  * Return a component with Storybook docs blocks in a standard order.
  * Accepts a JSON object as a prop representing component information
  */
-export function StoryDocs({ data }) {
-  // if it's a wc, it'll have a tag; otherwise, pass in component name
-  const componentName = data?.tag || data?.componentName;
-  const otherDocs = additionalDocs[componentName];
-  let updatedData;
-
-  if (otherDocs) {
-    updatedData = { ...data, ...otherDocs };
-  }
+export function StoryDocs({ componentName, data }) {
+  let componentData = data;
+
+  const component = componentName || data?.tag;
+  const componentDocs = additionalDocs?.[component];
 
-  const args = data?.props?.length > 0;
-  const events = data?.events?.length > 0 || data?.listeners?.length > 0;
+  if (componentData && componentDocs) {
+    componentData = { ...data, ...componentDocs };
+  }
 
-  const guidance = updatedData?.guidanceHref && updatedData?.guidanceName;
-  const maturity = updatedData?.maturityCategory && updatedData?.maturityLevel;
-  const description = updatedData?.description;
-  const isReactComponent = updatedData?.react;
+  const maturityCategory = componentDocs?.maturityCategory;
+  const maturityLevel = componentDocs?.maturityLevel;
+  const guidanceName = componentDocs?.guidanceName || componentName;
+  const guidanceHref = componentDocs?.guidanceHref;
+  const description = componentDocs?.description;
 
   return (
     <>
       <Title />
       <Subtitle />
-      {maturity && <MaturityScale data={updatedData} />}
-      {guidance && <Guidance data={updatedData} />}
-      {description && <ComponentDescription data={description} />}
-      {events && <CustomEventsDescription data={data} />}
-      <Description markdown={data.docs} />
+      <MaturityScale category={maturityCategory} level={maturityLevel} />
+      <Guidance href={guidanceHref} name={guidanceName} />
+      <ComponentDescription data={description} />
+      <CustomEventsDescription description={componentData} />
+      <Description markdown={data?.docs} />
       <Primary />
-      {(args || isReactComponent) && <ArgsTable story={PRIMARY_STORY} />}
+      <ArgsTable story={PRIMARY_STORY} />
```

Comment:
Does the args table still get shown correctly with the conditionals removed? I don't remember all the details, but I remember that `react` property being important in an earlier version.

Reply:
Everything appears to be working correctly (MDX doesn't display it, React and web components display it) without the need to check for props like we were previously doing in `args`. The `react` property was useful when we were performing a check to display the ArgsTable, but no longer needed now that checks are removed.